src/main/java/teammates/ui/pagedata/InstructorCourseEditPageData.java

```diff
@@ -225,6 +227,14 @@ private ElementTag createCancelEditInstructorButton(int panelIndex, boolean isDi
                                  isDisabled);
     }
 
+    private ElementTag createCancelAddInstructorButton() {
+        String buttonContent = "<span class=\"glyphicon glyphicon-remove\"></span> Cancel";
+        String buttonId = "addInstrCancelLink";
```

Comment:
I'm not sure if we should be mixing `link` and `button` namings.

Reply:
I agree the naming is a little confusing. However I realized all the other buttons (including the ones which involve cancelling) names it with a `Link` at the end too, so I followed this naming convention.

I'm concerned if naming this differently would affect how the other namings are interpreted.